bool call

---

bool    model_set
bool    solver_parameters_set
bool    environment_surface_set
bool    contact_sequence_set
bool    fully_initted
bool    solution_set